app/models/like.rb

```diff
@@ -0,0 +1,15 @@
+class Like < ApplicationRecord
+  # associatons
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # callbacks
+  after_save :update_post_likes_counter
+
+  # methods
+  private
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Have you ever considered using the `increment` Ruby method?

__HINT:__ 
```Ruby
def update_post_likes_counter
  post.increment!(:likes_counter)
end
```